documentation.md

```diff
@@ -44,7 +44,7 @@ An array of the plans you want to display. If not provided, the widget returns a
 
 The amount of time in between button animations in ms.
 
-- locale: `fr|en|es|it|de|nl|pt` [optional, default: en]
+- locale: `fr|fr-FR|es|es-ES|it|it-IT|de|de-DE|nl|nl-NL|nl-BE|pt|pt-PT|en` [optional, default: en]
```

Comment:
I think this is not the Widget job to check which locale is allowed or not,
We could inform that waited format is `locale-COUNTRY` or `locale` only (`fr-FR` or `fr` only) ... but we should not refuse an un-waited locale here (just fallback on default `en` if not found)

Reply:
The default en is specified but i'll update the doc to indeed say locale-COUNTRY instead of giving a few examples